term-utils/wall.c

```diff
@@ -56,11 +56,16 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
-#include <utmpx.h>
 #include <getopt.h>
 #include <sys/types.h>
 #include <grp.h>
 
+#if defined(HAVE_LIBSYSTEMD) && HAVE_DECL_SD_SESSION_GET_USERNAME == 1
```

Comment:
Shouldn't this be gated under `USE_SYSTEMD`?

Reply:
@t-8ch As already reported in #2101, `USE_SYSTEMD` is not defined with meson and it's not clear to me for what it should be good. It's always defined the same way as for HAVE_LIBSYSTEMD. What is the difference?